src/extensions/correlograms/CrossCorrelogramsView/CrossCorrelogramsView.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react'
```

Comment:
Here is the new cross correlograms view, modeled after the spike amplitudes view where the unit selection is in the left panel, with a splitter.